### PR TITLE
vm,metrics: note about behaviour across hypervisors

### PR DESCRIPTION
**Pull request by shwstppr**
----
📚 Documentation preview 📚: https://cloudstack-documentation--413.org.readthedocs.build/en/413/

### CODE REVIEW

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Review by rohityadavcloud — APPROVED** [review]

File: source/adminguide/virtual_machines.rst [hunk 1]
```diff
@@ -1505,6 +1505,16 @@ Instance disk statistics are shown in the Metrics tab in an individual volume vi
...
 
 
+
+.. note::
+   The metrics or statistics for VMs and VM disks in CloudStack depend on the
+   hypervisor plugin used for each hypervisor. The behavior can vary across
+   different hypervisors. For instance, with KVM, metrics are real-time
+   statistics provided by libvirt. In contrast, with VMware, the metrics are
+   averaged data based on the global configuration parameter
+   `vmware.stats.time.window`.
```
Comment (rohityadavcloud) [inline source/adminguide/virtual_machines.rst:1515]: @shwstppr Is this about any/all types of stats or just disk/network related?

Comment (shwstppr) [reply source/adminguide/virtual_machines.rst:1515]: @rohityadavcloud in my understanding this is true for all VM and disk stats

Comment (nvazquez) [inline source/adminguide/virtual_machines.rst:1515]: What about a second note for Vmware, advising that better results are observed by reducing the dynamic configuration value?

**Review by nvazquez — APPROVED** [review]

File: source/adminguide/virtual_machines.rst (hunk 1 above)
Comment (shwstppr) [reply source/adminguide/virtual_machines.rst]: @nvazquez I've added a statement

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Pull request merged**